edxadapt/static/html/student_view.html

```diff
@@ -0,0 +1,10 @@
+<div style="display: none;">
+Anonymous student  Id = {anonymous_student_id}
+</div>
+
+<div id="status_ok">
+    <i class="fa fa-check-circle" aria-hidden="true" style="color:green;"></i>
+</div>
+<div id="status_nok">
+    <i class="fa fa-exclamation-circle" aria-hidden="true" style="color:red;"></i> There was a technical issue while we were configuring your adaptive learning session. Please try to refresh this page or contact technical staff if problem persists.
+</div>
```

Comment:
since we have access to CSS, let's put the color into a CSS class:

HTML
```HTML
<div id="status_ok">
    <i class="fa fa-check-circle green" aria-hidden="true"></i>
</div>
<div id="status_nok">
    <i class="fa fa-exclamation-circle red" aria-hidden="true"></i> There was a technical issue while we were configuring your adaptive learning session. Please try to refresh this page or contact technical staff if problem persists.
</div>
```

CSS
```CSS
.green {
    color: green;
}
.red {
    color: red;
}
```